machines/api/working-with-machines-api.html.markerb

```diff
@@ -73,4 +73,10 @@ Typically, 2xx HTTP status codes denote successful operations, 4xx codes imply f
 | 401 | The API key used was missing or invalid. |
 | 404 | The resource was not found. |
 | 408 | Request timed out. |
-| 5xx | Indicates an error with Fly.io API servers. |
\ No newline at end of file
+| 5xx | Indicates an error with Fly.io API servers. |
+
+## Rate Limits
+
+Machine API rate limits apply _per-action_, _per-machine_. Currently, the limit is 1 request, per second, per action (i.e. Create Machine, Start Machine etc.)—short-term burstable up to 3 req/s, per action.
```

Comment:
Suggest using "burst limit" in place of "burstable" here and below. I feel like burst limit pairs with rate limit and is a familiar term. What do you think?